tools/vscode-azurewebpubsub/src/tree/service/ServiceItem.ts

```diff
@@ -16,6 +16,9 @@ import { ServicePropertiesItem } from "./ServicePropertiesItem";
 import { type ServiceModel } from "./ServiceModel";
 import { HubSettingsItem } from "../hubSettings/HubSettingsItem";
 
+let siteCacheLastUpdated = 0;
+const siteCache: Map<string, WebPubSubResource> = new Map<string, WebPubSubResource>();
```

Comment:
Put these two variables into the class `ServiceItem` looks better. In oreder to release today and I have no permission to update the branch, I'll update this in further PR.